core/shared/src/main/scala/io/udash/properties/ImmutableProperty.scala

```diff
@@ -52,7 +52,7 @@ private[properties] class ImmutableProperty[A](value: A) extends ReadablePropert
   override def transform[B](transformer: A => B): ReadableProperty[B] =
     new ImmutableProperty[B](transformer(value))
 
-  override def transformToSeq[B: PropertyCreator](transformer: A => Seq[B]): ReadableSeqProperty[B, ReadableProperty[B]] =
+  override def transformToSeq[B: PropertyCreator : DefaultValue](transformer: A => Seq[B]): ReadableSeqProperty[B, ReadableProperty[B]] =
```

Comment:
I don't get why this needs a `DefaultValue` - `transformer` should give us the whole sequence, no value-guessing needed.